app/views/ops/_diagnostics_database_tab.html.haml

```diff
@@ -118,7 +116,7 @@
           - submit = button_tag(_("Submit"), :class => "btn btn-primary", :alt => caption)
 
           = link_to(submit, {:action => 'db_gc_collection'},
-            "data-miq_sparkle_on" => true,
+            "data-miq_sparkle_off" => true,
```

Comment:
@GregP this change shouldn't be required if we use render_flash_and_stop_sparkle in controller.